.gitignore

```diff
@@ -39,4 +39,5 @@ nosetests.xml
 coverage.xml
 
 # Sphinx documentation
-docs/_build/
\ No newline at end of file
+docs/_build/
+.idea
```

Comment:
If possible, this kind of system-specific file should be added to a global gitignore on developers' own computers, rather than being added to the repo.

Reply:
removed